packages/assets-controllers/src/NftController.test.ts

```diff
@@ -385,6 +436,19 @@ describe('NftController', () => {
       );
     });
 
+    it('should error if tokenId is not a valid stringified decimal number', async function () {
+      const { nftController } = setupController();
+      const assetWithNumericTokenId = {
+        address: ERC721_NFT_ADDRESS,
+        tokenId: '123abc',
+      };
+
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore-next-line
```

Comment:
what are we ignoring about the next line?

Reply:
a type error, `watchNft` expects the tokenId in the first arg object to be a string. We're ignoring it precisely to test that the error is thrown for this case, since we currently don't enforce types in the extension where this is called.

Reply:
you could cast the fn to `any` or equivalent type instead:
`const erc721Result = (nftController.watchNft as any)(ERC721_NFT, type);`

Reply:
yep that would also work. Happy to do that if you think its better. Seems kindof the same to me